### DISCUSSION TITLE
GuardDuty detection bypass via cloudtrail

### DISCUSSION THREAD

**Discussion by megiddoa** [wiz-sec/open-cvdb#44] [General]
[https://www.cloudvulndb.org/guardduty-cloudtrail-bypass](url)

Please note that this GuardDuty detection bypass was remediated shortly after identified by Rhino Labs 2 years ago. GuardDuty will detect if this technique is used to disable CloudTrail logging.

**Comment by korniko98** [comment]
Hi, looking for a public reference for this led me to Scott's tweet [here](https://twitter.com/0xdabbad00/status/1258909131605307393), would you like to edit [the issue](https://github.com/wiz-sec/open-cvdb/blob/main/vulnerabilities/guardduty-cloudtrail-bypass.yaml) and submit a PR?

**Comment by korniko98** [comment]
Updated [the issue](https://www.cloudvulndb.org/guardduty-cloudtrail-bypass) to reflect the fix, using Scott's tweet as a reference - It's a shame there's no public archive of [SNS GuardDuty announcements](https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_sns.html)
